README.md

```diff
@@ -63,25 +65,30 @@ You will also need to install the appropriate build tools.
     mkdir c:\flowforge
     ```
 
-2. Download the Installer zip file from https://github.com/flowforge/installer/releases
+2. Download the latest [Installer zip file](https://github.com/flowforge/installer/releases/latest) into a temporary location.
```

Comment:
We can link directly to the file with

`https://github.com/flowforge/installer/releases/latest/download/flowforge-installer.zip`

Reply:
(once the next release is cut)

Reply:
Are we going to change this after Thursday?

Reply:
https://github.com/flowforge/installer/issues/61